coldfront/core/user/models.py

```diff
@@ -43,14 +46,41 @@ class EmailAddress(models.Model):
         }
     )
     is_verified = models.BooleanField(default=False)
-    is_primary = models.BooleanField(default=False)
+    is_primary = models.BooleanField(default=False,
+                                     help_text="Change is_primary status in list display.")
 
     class Meta:
         verbose_name = 'Email Address'
         verbose_name_plural = 'Email Addresses'
 
     def save(self, *args, **kwargs):
         self.email = self.email.lower()
+
+        # no old_primary_val if saving a new model. defaults to true for the logic
```

Comment:
There's a more elegant way of detecting prior existence that doesn't require adding a field. If the instance has a primary key (`if self.pk`), then it previously existed; otherwise, it's new. See `Project.save` for an example.

Reply:
Regarding changes to model fields, I believe you ran `makemigrations` and `migrate`.

My recommendation would be to [un-migrate](https://stackoverflow.com/a/32124113) this new migration, delete the migration file, remove the unneeded `new_email_flag`, and create a new migration file.

(Also, migration files should be renamed and committed so that other instances can run them.)